engine/src/main/java/org/hibernate/validator/internal/util/logging/Log.java

```diff
@@ -841,4 +841,10 @@ ValidationException getUnableToAccessMethodException(Lookup lookup, @FormatWith(
 	@LogMessage(level = DEBUG)
 	@Message(id = 240, value = "Constraint validator payload set to %1$s.")
 	void logConstraintValidatorPayload(Object payload);
+
+	@Message(id = 241, value = "The constraint validator %1$s is registered for the constraint %2$s but is defined with the constraint %3$s.")
```

Comment:
"registered" and "defined" may not be intuitive for everyone. Perhaps:

> Constraint @Foo references constraint validator type Bar, but this validator is defined for constraint type Qux

?

Reply:
Right, done.